src/helm/benchmark/presentation/summarize.py

```diff
@@ -389,6 +397,13 @@ def group_runs(self):
                 self.group_adapter_to_runs[group_name][adapter_spec].append(run)
                 self.group_scenario_adapter_to_runs[group_name][scenario_spec][adapter_spec].append(run)
 
+    def write_schema(self):
```

Comment:
In general, we have multiple schemas.  When we run summarize, we pick one.  Write this out so that the frontend can read the right one.